source/Assets/Plugins/Didomi/Tests/TextsTestsSuite.cs

```diff
@@ -0,0 +1,71 @@
+using System.Collections;
+using NUnit.Framework;
+using UnityEngine.TestTools;
+using IO.Didomi.SDK;
+using UnityEngine;
+using IO.Didomi.SDK.Events;
+
+/// <summary>
+/// Tests related to SDK texts
+/// </summary>
+public class TextsTestsSuite: DidomiBaseTests
```

Comment:
would maybe `TranslationsTestsSuite` or `LanguagesTestsSuite` be more accurate as a class name?

Reply:
ok for `TranslationsTestsSuite`